### PR TITLE
Add MSBuild timing Dev Inner Loop Tests

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

**Comment by DrewScoggins** [comment]
PTAL @arkalyanms @ladipro

File: src/scenarios/emptyconsoletemplateinnerloopmsbuild/src/Program.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (billwert) [inline src/scenarios/emptyconsoletemplateinnerloopmsbuild/src/Program.cs:1]: jamming this comment here for lack of a  better place - I don't think this should be `emptyconsoletempalteinnerloopmsbuild` if it is static code. I think generally we use `empty` to mean "having run `dotnet new`". Contrast `staticconsoletemplate` for example. Maybe this convention is already broken and I hadn't noticed before.

**Review by billwert — APPROVED** [review]
Going to approve this but please look at the one comment.

File: src/scenarios/emptyconsoletemplateinnerloopmsbuild/src/Program.cs (hunk 1 above)
Comment (DrewScoggins) [reply src/scenarios/emptyconsoletemplateinnerloopmsbuild/src/Program.cs:1]: This does run dotnet new, yeah? The last five lines of the pre.py makes it seem like that is what we are doing.

Comment (billwert) [reply src/scenarios/emptyconsoletemplateinnerloopmsbuild/src/Program.cs:1]: Oh this is the 'changed' source, I understand now. Forgot what context we were in.

**Pull request merged**